client/src/pages/Feed.js

```diff
@@ -277,13 +277,13 @@ const Feed = (props) => {
     }
   };
 
-  const handleChangeType = (e) => {
+  const handleChangeType = useCallback((e) => {
```

Comment:
Is this `useCallback` also added automatically?

Reply:
yes also automatically added